caps/capabilities_test.go

```diff
@@ -93,3 +99,20 @@ func (s *CapabilitySuite) TestNames(c *C) {
 	repo.Add(&Capability{"c", "label-c", FileType})
 	c.Assert(repo.Names(), DeepEquals, []string{"a", "b", "c"})
 }
+
+func (s *CapabilitySuite) TestString(c *C) {
+	cap := &Capability{"name", "label", FileType}
+	c.Assert(cap.String(), Equals, "name")
+}
+
+func (s *CapabilitySuite) TestAll(c *C) {
+	repo := NewRepository()
+	repo.Add(&Capability{"a", "label-a", FileType})
```

Comment:
You insert in order here, so the sorting is not tested implicitly here (and not explicit too AFAICS). So I would suggest to either test that sort works explicitly (or drop if its not important, I have no idea about that :)